services-custom/dynamodb-enhanced/src/test/java/software/amazon/awssdk/enhanced/dynamodb/converters/attribute/BinaryAttributeConvertersTest.java

```diff
@@ -56,4 +58,15 @@ public void sdkBytesSetAttributeConverter_ReturnsBSType() {
         SetAttributeConverter<Set<SdkBytes>> bytesSet = SetAttributeConverter.setConverter(SdkBytesAttributeConverter.create());
         assertThat(bytesSet.attributeValueType()).isEqualTo(AttributeValueType.BS);
     }
+
+    @Test
```

Comment:
Could you please also write Functional Test case with 
[FakeItemWithByteBufferKey.txt](https://github.com/aws/aws-sdk-java-v2/files/6245810/FakeItemWithByteBufferKey.txt)
 and then add this test case in CreateTableOperationTest.java as below

```
    @Test
    public void generateRequest_withByteBufferKey() {
        CreateTableOperation<FakeItemWithByteBufferKey> operation = CreateTableOperation.create(CreateTableEnhancedRequest.builder()
                .build());

        CreateTableRequest request = operation.generateRequest(FakeItemWithByteBufferKey.getTableSchema(),
                PRIMARY_CONTEXT,
                null);

        assertThat(request.tableName(), is(TABLE_NAME));
        assertThat(request.keySchema(), containsInAnyOrder(KeySchemaElement.builder()
                .attributeName("id")
                .keyType(HASH)
                .build()));

        assertThat(request.globalSecondaryIndexes(), is(empty()));
        assertThat(request.localSecondaryIndexes(), is(empty()));

        assertThat(request.attributeDefinitions(), containsInAnyOrder(
                AttributeDefinition.builder()
                        .attributeName("id")
                        .attributeType(ScalarAttributeType.B)
                        .build()));
    }
```

Reply:
Hi @joviegas ,

updated, thanks for preparing this so well.